docs/README.md

```diff
@@ -0,0 +1,12 @@
+# SQL binding extension for Azure Functions
+
+Azure SQL bindings in Azure Functions supports input and output bindings for the Azure SQL and SQL Server products.
```

Comment:
`Azure SQL bindings for Azure Functions adds input and output bindings for Azure SQL and SQL Server products.`

That reads clearer to me - @dzsquared thoughts?

Reply:
I'm ok with it either way - the text was likely taken from the docs, which I can update to match the wordsmithing